sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusProcessor.cs

```diff
@@ -163,6 +177,8 @@ public virtual bool IsClosed
         // deliberate usage of List instead of IList for faster enumeration and less allocations
         private readonly List<ReceiverManager> _receiverManagers = new List<ReceiverManager>();
         private readonly ServiceBusSessionProcessor _sessionProcessor;
+        private volatile List<(Task, CancellationTokenSource, CancellationTokenSource)> _tasks;
```

Comment:
I'm curious why we would need `volatile` here;   If the list is created when `RunReceiveTaskAsync` is called during `StartProcessingAsync` and is stable until `StopProcessingAsync` is called,  is it possible that the instance changes?   Could we new things up here and just clear the instance at stop time?